src/Microsoft.Azure.SignalR.Common/ClientInvocation/IRoutedClientResultsManager.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.SignalR.Protocol;
+
+namespace Microsoft.Azure.SignalR
+{
+    internal interface IRoutedClientResultsManager
+    {
+        Task<object> AddRoutedInvocation(string connectionId, string invocationId, string callerServerId, CancellationToken cancellationToken);
+
+        bool TryCompleteResult(string connectionId, CompletionMessage message);
+
+        bool TryGetRoutedInvocation(string invocationId, out RoutedInvocation routedInvocation);
+
+        bool TryGetInvocationReturnType(string invocationId, out Type type);
+
+        void CleanupInvocations(string instanceId);
```

Comment:
Routed server has client connection status. This method is not needed, or we may leverage `TryCompleteResult()` to  simply cleanup by `connectionId` regarding route server finds client disconnect?